runtime/precompiles/src/lib.rs

```diff
@@ -67,12 +67,3 @@ pub type MoonbeamPrecompiles<Runtime> =
 	// Bn128Pairing,
 	Dispatch<Runtime>,
 );
-
-
-#[cfg(test)]
-mod tests {
-	#[test]
-	fn it_works() {
-		assert_eq!(2 + 2, 4);
-	}
-}
```

Comment:
:eyes:

Reply:
This comes from  `cargo init --lib`. I don't know when it was added, but it seems like a worthless test, so I'm fine to get rid of it.